build.psm1

```diff
@@ -407,12 +405,18 @@ Fix steps:
         return
     }
 
+    # Set $RuntimeRID as Win7-x64 or Win7-x86 so the package works on all supported Windows OS versions.
+    switch ($Runtime) {
+        'win-x64' { $RuntimeRID = 'win7-x64' }
+        'win-x86' { $RuntimeRID = 'win7-x86' }
+    }
+
     # set output options
     $OptionsArguments = @{
         CrossGen=$CrossGen
         Output=$Output
         FullCLR=$FullCLR
-        Runtime=$Runtime
+        Runtime=$RuntimeRID
```

Comment:
@iSazonov we have to use win7-* to make sure the package works on all supported Windows OS versions. It would be weird for cmdline usage to specify win7-x86 or win7-x64, hence I changed it to win-x64 and win-x86. The script changes it to win7-*.